test/installation_node/install from binary

```diff
@@ -12,7 +12,7 @@ NVM_TEST_VERSION=v0.10.7
 [ -e ../../$NVM_TEST_VERSION ] && rm -R ../../$NVM_TEST_VERSION
 
 # Install from binary
-nvm install $NVM_TEST_VERSION || die "install $NVM_TEST_VERSION failed"
+nvm install -b $NVM_TEST_VERSION || die "install $NVM_TEST_VERSION failed"
```

Comment:
it'd probably be good to duplicate this test file, so we have tests for this both with and without the `-b` flag?

Reply:
Got it :+1: